src/main/java/org/kohsuke/github/internal/EnumUtils.java

```diff
@@ -49,7 +49,7 @@
      */
     public static <E extends Enum<E>> E getEnumOrDefault(Class<E> enumClass, String value, E defaultEnum) {
         try {
-            return Enum.valueOf(enumClass, value);
+            return Enum.valueOf(enumClass, value.toUpperCase(Locale.ROOT));
```

Comment:
I actually don't want to do this.  The values are case-sensitive on purpose.  I'm open to discussing changing this but not as part of this PR.